src/core/operations_constructor.jl

```diff
@@ -17,14 +23,14 @@ function _pass_abstract_jump(optimizer::Union{Nothing,JuMP.OptimizerFactory}; kw
     end
 
     JuMPmodel = JuMP.Model(optimizer)
-    PJ.enable_parameters(JuMPmodel)
+    parameters ? PJ.enable_parameters(JuMPmodel) : return JuMPmodel
```

Comment:
Here I would not use a ternary operator.

if parameters
    PJ.enable_parameters(JuMPmodel)
end

return JuMPmodel